web/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev --turbopack",
-    "build": "next build",
+    "dev": "NODE_ENV=development next dev --turbopack",
+    "build": "NODE_ENV=production next build",
```

Comment:
I thought NextJS was able to pick up the `NODE_ENV` automatically based on whether you are running `npm run dev` vs. `npm run build`, but maybe I am mistaken?